editor/src/components/inspector/sections/comment-section.tsx

```diff
@@ -161,12 +160,16 @@ const ThreadPreview = React.memo(({ thread }: ThreadPreviewProps) => {
     [resolveThread, dispatch, thread],
   )
 
+  const collabs = useStorage((storage) => storage.collaborators)
+
   const comment = thread.comments[0]
   if (comment == null) {
     return null
   }
 
   const repliesCount = thread.comments.length - 1
+  const user = collabs[comment.userId]
```

Comment:
Worth doing to make sure undefined's are handled:
```suggestion
  const user = collabs.at(comment.userId)
```

Reply:
that's not an array tho! it's an object!